src/platform_impl/windows/event_loop.rs

```diff
@@ -1383,7 +1396,6 @@ unsafe fn public_window_callback_inner<T: 'static>(
                 let mut w = userdata.window_state_lock();
                 w.set_window_flags_in_place(|f| f.set(WindowFlags::MINIMIZED, true));
             }
-            // Send `WindowEvent::Minimized` here if we decide to implement one
```

Comment:
Using `WM_SIZE` instead, since sending a Minimized even here would only trigger for window header buttons, and not a shortcut.